.ci/build-n-run.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+function build_example_code()
+{   
+    make -C examples/ || exit 1;
+}
+
+function gen_test_file_list()
```

Comment:
Alternatively, you can create a function such as `run_mod`, which accepts one parameter as the specified Linux kernel module. Then, use that function here.

Reply:
Sorry, I don't understand your meaning very well.
So I guess `run_mod` is supposed to run kernel module with accepted parameter only. (Without any string modification)
And I don't know why put it in `gen_test_file_list()`?
Can you give further explanations?

Reply:
It would better if you can implement another function which returns the filtered list of available Linux kernel modules, which can be then used with the `for` statement whose body consists of the invocation of `run_mod`.

That is,
```bash
for mod in $(mod_list)
do
    run_mod $mod
done
```

Reply:
See https://www.hostinger.com/tutorials/bash-for-loop-guide-and-examples/

Reply:
It is not elegant to rely on the function `gen_test_file_list` though. It does not make sense because you can filter out the non-working modules with the UNIX commands in combination.